data.js

```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-02-23",
+
+        },
+        description: {
+            content: "The World Rescource Institution (WRI) is a big and expierenced environmental NGO. Their approach is to measure their work. The motto is 'count it, change it, scale it'. They work not just work on climate issues but also on the topics of energy, food, forests, water, cities and water.",
+            updatedBy: "Andre", updatedAt: "2020-02-23",
+        },
+        
+        actions: {
+            content: "Examples from [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities in Africa, Asia and Latain America to develope more sustainable urban mobility."+ 
+                     "\n- Lobbyied for the Paris Agreement."+
+                     "\n- Helped to implement first binding environmental laws in 16 South American and Caribbean countries."+
+                     "\nRead (more)[https://www.wri.org/our-work/outcomes] stories of success.",
+            updatedBy: "Andre", updatedAt: "2020-02-23",
+
+        },
+        activeCountries: {
+            content: "They have [offices globally](https://www.wri.org/annualreport/2018-19), their headquarter is in Washington, DC.",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
+        },
+        yearlyIncome: {
+            content: "[~ $ 111M in 2018](https://wriorg.s3.amazonaws.com/s3fs-public/uploads/wri-2018-19-annual-report_0.pdf#page=48)",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
+        },
+        members: {
+            content: "over [1000 scientists and staff members](https://www.wri.org/about/experts-staff)",
```

Comment:
Uppercase `O`. Please double-check your PRs everytime. Reviewers tend to miss a lot, too.

Reply:
@yeah800 I updated crisisfighters.org with your changes, including the `O`. We can close this now and have the new content online :)